spec/index.html

```diff
@@ -5669,248 +5670,267 @@ <h5>NOT IN</h5>
+                    <td><code>sameTerm(&lt;http://example/&gt;, &lt;https://example/&gt;)</code></td>
+                    <td>false</td>
+                  </tr>
+                  <tr>
+                    <td><code>sameTerm("abc", "abc")</code></td>
+                    <td>true</td>
+                  </tr>
+                  <tr>
+                    <td><code>sameTerm("abc"@en, "abc")</code></td>
+                    <td>false</td>
+                  </tr>
+                  <tr>
+                    <td><code>sameTerm("abc"@en, "abc"@EN)</code></td>
+                    <td>true</td>
+                  </tr>
+                  <tr>
+                    <td><code>sameTerm("abc"@en--rtl, "abc"@en)</code></td>
+                    <td>true</td>
+                  </tr>
+                  <tr>
+                    <td><code>sameTerm(2, 2.0)</code></td>
+                    <td>false</td>
+                  </tr>
+                  <tr>
+                    <td><code>sameTerm(2, "2"^^xsd:integer)</code></td>
+                    <td>true</td>
+                  </tr>
+                  <tr>
+                    <td><code>sameTerm(2, "02"^^xsd:integer)</code></td>
+                    <td>false</td>
+                  </tr>
+              </table>
+            </div>
+
+          </section>
+          <section id="func-RDFterm-equal">
+            <h5>RDFterm-equal</h5>
+            <pre class="prototype nohighlight">
+              <span class="return">xsd:boolean</span> <span class="operator" style="text-transform: none;">RDFterm-equal</span> (<span class="type"><span class="type RDFterm">RDF term</span></span> <span class="name">term1</span>, <span class="type"><span class="type RDFterm">RDF term</span></span> <span class="name">term2</span>)
+            </pre>
+
+            <p>This function cannot be used directly in expressions. The purpose
+              of this function is to define the semantics of the "=" operator when applied to
+              two RDF terms that do not fall into any of the other, more concrete cases
+              covered in the operator mapping table in Section
+              <a href="#OperatorMapping" class="sectionRef"></a>.
+            </p>
+
+            <p class="ednote" style="background-color: #EEE">
+              This will be renamed <code>sameValue</code></p>
+
+            <ol>
+              <li>If one or both arguments are known to be ill-typed, 
+                then produce a type error.
+              </li>
+              <li>Returns TRUE if <code>term1</code> and <code>term2</code>
+                are <em>equal</em> RDF terms, as defined below.
+              </li>
+              <li>If the SPARQL processor can determine the values of both terms
+                and it can determine the values are equal, then return TRUE.
+              </li>
+              <li>If the SPARQL processor can determine that the values of the terms
+                can not be equal, the return FALSE.</li>
```

Comment:
Are you assuming that the two terms are literals? If yes, why is it a valid assumption to be made at this point?

Reply:
Added "if <code>term1</code> and <code>term2</code> are both literals" ... as per @tpt suggestion.

But this needs revising again (not in PR yet) for triple terms.

Only literal-literal sameValue can be extended but it should apply recursively.

IRIs and blank nodes have a loose (not defined) sense of "value" through same term. 

Maybe the Operator Dispatch table could have "IRI = IRI", "blank node" = "blank node" and "triple term = triple term" added. Then `sameValue` is defined for literals.

Reply:
Section revised.

Also added  "IRI = IRI", "blank node" = "blank node" and "triple term = triple term" to the Operator Dispatch table.